src/components/NcSettingsSection/NcSettingsSection.vue

```diff
@@ -154,7 +154,7 @@ $maxWidth: 900px;
 		// make sure to properly align the icon with the text
 		margin: -$icon-margin;
 		margin-left: 0;
-		opacity: $opacity_normal;
+		color: var(--color-text-lighter);
```

Comment:
By the way, this changes the help icon, not the text, and as a result, removes the hover effect from the help icon link. Is it intended?

Reply:
opacity is on general case bad choice. Hover effect itself is not as important but if this hover effect is not accessible - there is a11y problem again ;(

Reply:
Then maybe add a hover effect by making it `--color-main-text` on hover.